### PR TITLE
Add with-commit-hash and rename project

### PR DESCRIPTION
**Pull request by kbaba1001**
* use edn instead of manifest
* I renamed project: from `clj-build-date` to `metabill`
  *  Could you change the project name?
* I added a function `with-commit-hash`, which is able to get the commit hash building jar

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,24 @@
-# clj-build-date
+# clj-manifest-reader
```
Comment (kbaba1001) [inline README.md]: rename idea:

* `lein-metabill` ... a leiningen plugin to set `manifest` in `project.clj`
* `metabill` ... a library to load metadata from `manifest`

**Comment by federkasten** [comment]
@kbaba1001 Thanks for your contribution. I will change the PR status to Draft, as I think you would like to rename the project again and implement the leiningen plugin.

File: README.md (hunk 1 above)
Comment (kbaba1001) [reply README.md]: I tried using https://github.com/technomancy/leiningen/blob/master/doc/PLUGINS.md#profiles but it didn't work 💢

File: README.md [hunk 2]
```diff
@@ -1,40 +1,50 @@
...
 
-A minimal library to read build timestamp from JAR/WAR manifests.
-
-Useful for prevent browsers from caching old js and css files.
+A minimal library to read build info.
 
 ## Prepare
 
-Add this line in your `project.clj`.
+Put `[lein-metabill "0.1.0"]` into the `:plugins` vector of your project.clj.
```
Comment (notake) [inline README.md]:
```suggestion
Put `[xcoo/metabill "0.1.0"]` into the `:dependencies` vector and `[xcoo/lein-metabill "0.1.0"]` into the `:plugins` vector of your project.clj.
```

File: README.md [hunk 3]
```diff
@@ -1,40 +1,50 @@
...
+    (include-css (with-build-date "/css/main.css"))]
    [:body
     [:div#app]
-    (include-js (with-build-date req "/js/main.js"))]))
+    (include-js (with-build-date "/js/main.js"))]))
 ```
 
 ![network](https://raw.githubusercontent.com/xcoo/clj-build-date/master/img/network.png)
 
+### with-commit-hash
```
Comment (notake) [inline README.md]:
```suggestion
### commit-hash
```

**Review by notake — APPROVED** [review]
I checked this plugin locally. LGTM :+1:
The source code is fine, but please make some changes to the documentation.

**Review by notake — APPROVED** [review]

**Review by federkasten — APPROVED** [review]

**Pull request merged**